extensions/amp-story/1.0/test/test-amp-story-bookend.js

```diff
@@ -268,6 +271,300 @@ describes.realWin('amp-story-bookend', {
     });
   });
 
+  it('should add amp-to-amp linking to individual cta links when ' +
+      'specified in the JSON config', () => {
+    const userJson = {
+      'bookend-version': 'v1.0',
+      'share-providers': [
+        'email',
+        {'provider': 'facebook', 'app_id': '254325784911610'},
+        'whatsapp',
+      ],
+      'components': [
+        {
+          'type': 'cta-link',
+          'links': [
+            {
+              'text': 'buttonA',
+              'url': 'google.com',
+              'ampdoc': true,
```

Comment:
Can we use "amphtml" instead of "ampdoc", which is overloaded e.g. ampdoc-impl.js?

Reply:
Should this be `"amphtml": true` or `"rel": "amphtml"`?

Reply:
I was thinking `amphtml: true`.